Send cover flag

agpsl opened the pull request:
If the environment variable `SEND_ALBUM_COVER` is set to `false`, only the audio is sent

agpsl committed:
Cover flag

Create a flag on token.env (no better place to do that imo) to allow
users to chose if they want the album cover and metadata to be sent or
not

agpsl committed:
Rookie mistake smh

kernoeb commented:
Interesting, thanks!
I'm on vacation right now, from my phone, I'll check this asap :)

agpsl committed:
Doc for the SEND_ALBUM_COVER flag

agpsl committed:
Make TORY, TYER and TDAT td3 tags optional

The song wasn't being sent if one of the tags wasn't present. Make it
optional.

kernoeb commented:
Hi, is there a particular reason you removed the `TPUB` tag in the `maketag` part ?

kernoeb committed:
chore: lint

agpsl committed:
Reintroduce TPUB

agpsl commented:
No reason other than a mistake. It's fixed now.

kernoeb commented:
Thanks! Sorry for the long wait before merging
